noxfile.py

```diff
@@ -0,0 +1,169 @@
+import os
+from contextlib import contextmanager
+from inspect import cleandoc
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import nox
+from pyodbc import connect
+
+PROJECT_ROOT = Path(__file__).parent
+
+# default actions to be run if nothing is explicitly specified with the -s option
+nox.options.sessions = ["verify"]
+
+
+class Settings:
+    ITDE = PROJECT_ROOT / ".." / "integration-test-docker-environment"
+    ODBC_DRIVER = PROJECT_ROOT / "driver" / "libexaodbc-uo2214lv1.so"
+    CONNECTORS = ("pyodbc", "turbodbc")
+    ENVIRONMENT_NAME = "test"
+    DB_PORT = 8888
+    BUCKETFS_PORT = 6666
+
+
+ODBCINST_INI_TEMPLATE = (
+    cleandoc(
+    """
+    [ODBC]
+    #Trace = yes
+    #TraceFile =~/odbc.trace
+
+    [EXAODBC]
+    #Driver location will be appended in build environment:
+    DRIVER={driver}
+    """)
+    + "\n"
+)
+
+
+def find_session_runner(session, name):
```

Comment:
I don't know nox that well, I think, I need a bit explanation here

Reply:
:heavy_check_mark: